### PR TITLE
Use Scala 2.12.10

### PR DESCRIPTION
**Pull request by aeons**

### CODE REVIEW

File: project/plugins.sbt [hunk 1]
```diff
@@ -1,4 +1,4 @@
-resolvers += "jgit-repo" at "http://download.eclipse.org/jgit/maven"
+resolvers += "jgit-repo" at "https://download.eclipse.org/jgit/maven"
```
[the remark this answers is not in the document]
Comment (aeons) [reply project/plugins.sbt]: I tried googling a bit, and it seems there was a problem that jgit was not on maven at some point. It looks like its on maven now, but I'm not sure what needs it.

**Comment by aeons** [comment]
Metals is tracking 2.12.10 support here: https://github.com/scalameta/metals/pull/913

**Comment by rossabaker** [comment]
Okay, let's merge this once that metals issue is resolved.

**Comment by aeons** [comment]
A metals release with support for 2.12.10 is out now.

**Pull request merged**